### ISSUE TITLE
Add documentation or link on how to use the catalog

### ISSUE THREAD

**Issue by data-henrik** [open-toolchain/tekton-catalog#163]
The README does not contain any instructions on how to (re)use the assets in this catalog. It would be great to have at least some links to outside documentation on how to install and use an asset.

The linked Continuous Delivery service and its Tekton section do not have any discussion of this catalog.

**Comment by ejodet** [comment]
@data-henrik - well - the README https://github.com/open-toolchain/tekton-catalog#tasks section
lists and links all the tasks in the catalog.
Then, each task has a readme where setup is detailed with screen shots - ex: https://github.com/open-toolchain/tekton-catalog/tree/master/git/sample

Please elaborate on what is missing here - or what do you expect.

Regarding "The linked Continuous Delivery service and its Tekton section do not have any discussion of this catalog."
Good point, I will open a doc issue and work on the content

**Comment by data-henrik** [comment]
When on https://github.com/open-toolchain/tekton-catalog#tasks I click on, e.g., icr-containerize, I get to https://github.com/open-toolchain/tekton-catalog/blob/master/container-registry/README.md#icr-containerize. There is no link to any sample, nothing on installation.

Do you have documentation on "best practices" or a tutorial on how to use this asset catalog? I have opened Usabilla feedback on the CD docs, too.

**Comment by data-henrik** [comment]
I navigated to one of the subdirectories and found https://github.com/open-toolchain/tekton-catalog/tree/master/container-registry#install-the-tasks:

> Install the Tasks
> 
>     Add a github integration to your toolchain with the repository containing the tasks (https://github.com/open-toolchain/tekton-catalog)
>     Add this github integration to the Definitions tab of your Continuous Delivery tekton pipeline, with the Path set to container-registry
> 

Do you have a template or sample on how to add this catalog to a toolchain / pipeline definition? Something to put into a ".bluemix" directory when not using the UI?

**Comment by ejodet** [comment]
@data-henrik based on our discussion, I noticed that the first tasks we uploaded to the catalog
all contain a /sample folder where a readme details the setup instructions you expected,
as well as a working pipeline and listener.

On the other hand, most recent tasks don't have such folder and instructions.
So we need to review / add what's missing.

@jauninb ^^
I also noticed that, for example, the signing/dct task's readme contains very little information
(nothing on the params....)

Regarding the ibmcloud doc, I created an issue https://github.ibm.com/org-ids/ua-issues/issues/1382
and made a PR so as to links to:

- https://github.com/open-toolchain/tekton-catalog
- https://www.ibm.com/cloud/blog/ibm-cloud-continuous-delivery-t

ekton-pipelines-tools-and-resources
